app/models/collection.rb

```diff
@@ -2,16 +2,16 @@ class Collection < ApplicationRecord
   include Filterable
   include WorksOwner
 
-  has_attached_file :icon,
-  styles: { standard: "100x100>" },
-  url: "/system/:class/:attachment/:id/:style/:basename.:extension",
-  path: %w(staging production).include?(Rails.env) ? ":class/:attachment/:id/:style.:extension" : ":rails_root/public:url",
-  storage: %w(staging production).include?(Rails.env) ? :s3 : :filesystem,
-  s3_protocol: "https",
-  default_url: "/images/skins/iconsets/default/icon_collection.png"
+  has_one_attached :icon do |attachable|
+    attachable.variant(:standard, resize_to_fill: [100, nil])
```

Comment:
It's been a bit since I wrote this particular bit, but I vaguely remember an SO answer that was related to the trailing `>`. But according to https://stackoverflow.com/a/26272192 (and a couple other answers) whatever I interpreted wasn't right

Reply:
Thanks for the link!

That doc raises another question: Whether `resize_to_fill` is the correct method. Based on the [description in ImageMagick](http://www.imagemagick.org/script/command-line-processing.php#geometry):
>  Use > to shrink an image only if its dimension(s) are larger than the corresponding width and/or height arguments.

I think the method to keep this behaviour would be [resize_to_limit](https://github.com/janko/image_processing/blob/master/doc/vips.md#resize_to_limit):

> Downsizes the image to fit within the specified dimensions while retaining the original aspect ratio. Will only resize the image if it's larger than the specified dimensions.

Reply:
Good call! (And thanks for the docs link, I've had trouble tracking down robust documentation for the new image resizing implementation)